var/spack/repos/builtin/packages/py-basemap/package.py

```diff
@@ -35,87 +34,11 @@ class PyBasemap(PythonPackage):
 
     version('1.0.7', '48c0557ced9e2c6e440b28b3caff2de8')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
Please add a comment for this line that (1) gives a brief explanation and (2) links to https://github.com/LLNL/spack/issues/3813